### DISCUSSION TITLE
Welcome to rtorrent Discussions!

### DISCUSSION THREAD

**Discussion by rakshasa** [rakshasa/rtorrent#1068] [General]
## 👋 Welcome!

Discussions can now happen.

**Comment by lilws** [comment]
rtorrent running in 0.9.8 for a long time, I wonder when will it become stable? Some tracker is very strict with beta version. They've just banned the beta version without any good reason, since 0.9.8 is very stable so far.

**Reply by rakshasa** [reply]
Marked 0.9.8 as stable.
